base/model/src/test/java/org/eclipse/ditto/base/model/common/ILikeHelperTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.base.model.common;
+
+import java.util.regex.Pattern;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public final class ILikeHelperTest {
```

Comment:
I would propose to move this test to the `LikeHelperTests` - so that this test tests with `Pattern.CASE_INSENSITIVE` (which you added) and without (already existing).

Then please also remove the `ILikeHelperTest`.